VEDA_OS01/tests/test_views.py

```diff
@@ -0,0 +1,232 @@
+""" Views tests """
+import json
+import responses
+
+from ddt import data, ddt, unpack
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from mock import patch
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from VEDA_OS01.transcripts import CIELO24_LOGIN_URL
+from VEDA_OS01.models import TranscriptCredentials, TranscriptProvider
+
+
+@ddt
+class TranscriptCredentialsTest(APITestCase):
+    """
+    Transcript credentials tests
+    """
+    def setUp(self):
+        """
+        Tests setup.
+        """
+        super(TranscriptCredentialsTest, self).setUp()
+        self.url = reverse('transcript_credentials')
+        self.user = User.objects.create_user('test_user', 'test@user.com', 'test')
+        self.client.login(username=self.user.username, password='test')
+
+    def test_transcript_credentials_not_allowed(self):
```

Comment:
better if we can rename it to `test_transcript_credentials_get_not_allowed` or `test_get_transcript_credentials_not_allowed`